components/identity-mgt/org.wso2.carbon.identity.mgt.ui/src/main/resources/web/identity-mgt/forgot_password_step1.jsp

```diff
@@ -151,7 +152,7 @@
         </table>
 
         <script type="text/javascript">
-              showCaptcha('<%=captchaImageUrl%>');
+              showCaptcha('<%=Encode.forJavaScriptAttribute(captchaImageUrl)%>');
```

Comment:
This should be Encode.forJavascritptBlock since the function was invoked within a script block